src/components/Indicator/Indicator.scss

```diff
@@ -17,17 +16,38 @@ $base-position: rem(-4px);
   }
 }
 
-.pulseIndicator::before {
+.Indicator-pulse::before {
```

Comment:
Since the filename is called Indicator, these can all just be: `.pulse`, `.positive`, `.neutral`, `.negative. 

This will end up generating: `.Polaris-Indicator--pulse` instead of  `.Polaris-Indicator__Indicator--pulse`.